src/software/thunderscope/thunderscope.py

```diff
@@ -429,10 +430,17 @@ def configure_full_system_layout(
         playinfo_dock = Dock("Play Info")
         playinfo_dock.addWidget(widgets["playinfo_widget"])
 
+        widgets["refereeinfo_widget"] = self.setup_referee_info(
+            full_system_proto_unix_io
+        )
+        refereeinfo_dock = Dock("Referee Info")
+        refereeinfo_dock.addWidget(widgets["refereeinfo_widget"])
+
         dock_area.addDock(field_dock)
         dock_area.addDock(log_dock, "left", field_dock)
         dock_area.addDock(parameter_dock, "above", log_dock)
         dock_area.addDock(playinfo_dock, "bottom", field_dock)
+        dock_area.addDock(refereeinfo_dock, "bottom", field_dock)
         dock_area.addDock(performance_dock, "right", playinfo_dock)
```

Comment:
Lets update the ordering here so the new widget is added as a tab next to the playinfo widget. Kind of unrelated, but this also makes the Log widget as the default tab, instead of the Parameters widget.
```suggestion
        dock_area.addDock(field_dock)
        dock_area.addDock(parameter_dock, "left", field_dock)
        dock_area.addDock(log_dock, "above", parameter_dock)
        dock_area.addDock(refereeinfo_dock, "bottom", field_dock)
        dock_area.addDock(playinfo_dock, "above", refereeinfo_dock)
        dock_area.addDock(performance_dock, "right", playinfo_dock)
```